geode-core/src/main/java/org/apache/geode/internal/logging/log4j/FastLogger.java

```diff
@@ -47,30 +51,333 @@ public static void setDelegating(final boolean newValue) {
   }
 
   @Override
-  public boolean isDebugEnabled() {
-    return delegating && super.isDebugEnabled();
+  public boolean isEnabled(Level level, Marker marker, Message message, Throwable t) {
+    if (level.intLevel() > DEFAULT_LEVEL.intLevel() && !delegating) {
```

Comment:
the expression (level.intLevel() > DEFAULT_LEVEL.intLevel() && !delegating) is reused many times. Consider refactoring it into a private method.

Reply:
Also it is not clear to me that DEFAULT_LEVEL is a win over Level.INFO which is also a constant. If it could refer to the place that actually defines the default level then it would be a clear win.